packages/matter.js/src/protocol/interaction/InteractionServer.ts

```diff
@@ -46,6 +46,20 @@ export const INTERACTION_PROTOCOL_ID = 0x0001;
 
 const logger = Logger.get("InteractionProtocol");
 
+function getEnabledBits(bitmask: number): number[] {
```

Comment:
if you do not encoe the feature data above you do not need to do this here :-)

Reply:
If you have a way of encoding something more basic than number[] I am all ears.  The Attributes don't have scope of the Features and I didn't think it appropriate to make the dependency.  Dig deeper if you think there is a better solution.